pkg/controller/portworxdiag/portworxdiag.go

```diff
@@ -292,9 +371,49 @@ func shouldPodBeOnNode(nodeID string, nodeName string, nodes []v1.Node, diag *di
 	return true // All labels in selector passed
 }
 
+func (c *Controller) getNodesToHavePods(nodes *v1.NodeList, diag *diagv1.PortworxDiag, stc *corev1.StorageCluster, nodeIDToNodeName map[string]string, nodeIDToStatus map[string]string) (map[string]bool, error) {
```

Comment:
nit: Can we add a comment above this line? Or maybe rename to something more descriptive? Perhaps `nodesSelectedForPods` or get `getSelectedNodesForPods`

I was confused initially what this function was doing without reading the function.